e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -579,15 +579,12 @@ Remove-Item -r $selfSignedCerts
 
 Write-Host("`nWriting secrets to KeyVault $keyVaultName")
 az keyvault set-policy -g $ResourceGroup --name $keyVaultName --object-id $userObjectId --secret-permissions delete get list set --output none
-az keyvault secret set --vault-name $keyVaultName --name "IOTHUB-CONN-STRING-CSHARP" --value $iotHubConnectionString --output none
```

Comment:
We might be using this in our samples repo as well, worth checking there.

Reply:
https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/vsts/vsts.yaml#L52
https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/vsts/vsts.yaml#L94

Reply:
Thanks!